### ISSUE TITLE
Replacing an html-safe empty string at the end of an each causes a rendering error

### ISSUE THREAD

**Issue by mongoose700** [emberjs/ember.js#16314]
The error is triggered when it tries to replace the html-safe empty string in the `each` with anything. It also requires that within the `each`, the html-safe empty string is the last thing to be displayed.

The error triggered is
```
Uncaught TypeError: Cannot read property 'nextSibling' of null
```
in
```
DOMChanges.prototype.insertAfter = function insertAfter(element, node, reference) {
  this.insertBefore(element, node, reference.nextSibling);
};
```

It is reproduced here:
https://ember-twiddle.com/845f172f2878679fcba7fd6ec62ca48f

**Comment by Serabe** [comment]
The problem is with the `~` in:

```hbs
{{#each items as |item|}}
  {{item~}}
{{/each}}
```

**Comment by mongoose700** [comment]
Yes, the error does not happen when the `~` is removed. That's why I specified "It also requires that within the `each`, the html-safe empty string is the last thing to be displayed."

**Comment by chancancode** [comment]
Sorry it took so long, this is fixed in 3.4.3 (#17003)
